VALUES.md

```diff
@@ -3,6 +3,9 @@
 We want to make sure every member has a shared understanding of the goals and
 values we hold as a team:
 
+- Istio may not endanger a consuming organization's objectives or, through
```

Comment:
What is the consuming org here?  Is this vendors building solutions on top of istio or users?

Reply:
reworded.